src/components/PostGrid.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import Avatar from './Avatar';
+import { updateVotesByRepo } from '../lib/supabase';
+import { getRepoLink } from '../lib/github';
+import useSupabaseAuth from '../hooks/useSupabaseAuth';
+import { User } from "@supabase/supabase-js";
+import { FaArrowAltCircleUp } from "react-icons/fa";
+
+export declare interface PostGridProps {
+  data: any;
```

Comment:
I am not a strong TypeScript user, but the use of "any" here seems to be raising some flags.